.github/workflows/ci.yml

```diff
@@ -133,15 +129,15 @@ jobs:
         run: |
           ./scripts/kubewarden-load-policies.sh \
               --policies policies.tar.gz \
-              --policies-list e2e-tests/airgap/policies.txt \
+              --policies-list tests/data/airgap/policies.txt \
```

Comment:
Given that now we use testcontainers, we could move this airgap test out of GHA and into local rust tests.
But this is not a merge blocker for me.

Reply:
@viccuad I will add an issue about this